src/RazorSdk/Tasks/Microsoft.NET.Sdk.Razor.Tasks.csproj

```diff
@@ -0,0 +1,118 @@
+<Project>
+
+  <PropertyGroup>
+    <RazorSdkRoot>$(RepoRoot)\src\RazorSdk\</RazorSdkRoot>
+    <PackageId>Microsoft.NET.Sdk.Razor</PackageId>
+    <OutDirName>$(Configuration)\Sdks\$(PackageId)\tasks</OutDirName>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <!-- MSBuild Task DLLs need to be versioned with every build -->
+    <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
+
+    <OutputPath>$(BaseOutputPath)</OutputPath>
+    <IsPackable>true</IsPackable>
+    <PackageLayoutOutputPath>$(ArtifactsBinDir)$(Configuration)\Sdks\$(PackageId)\</PackageLayoutOutputPath>
+    <NoDefaultExcludes>true</NoDefaultExcludes>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Description>SDK for building and publishing Razor applications.</Description>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.NET.Sdk.Razor</RootNamespace>
+    <TargetFrameworks>$(SdkTargetFramework);net46</TargetFrameworks>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build.Framework" Version="15.8.166" ExcludeAssets="Runtime" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.8.166" ExcludeAssets="Runtime" />
+    <PackageReference Include="System.Reflection.Metadata"  Version="$(SystemReflectionMetadataVersion)" Condition="'$(TargetFramework)'=='net46'" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.Extensions"
+      Version="5.0.1"
+      Condition="'$(TargetFramework)'!='net46'" 
+      ReferenceOutputAssembly="false"
+      SkipGetTargetFrameworkProperties="true"
+      UndefineProperties="TargetFramework;TargetFrameworks" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <AdditionalContent Include="$(RazorSdkRoot)Targets\**\*.*">
+      <Pack>true</Pack>
+      <PackagePath>targets</PackagePath>
+    </AdditionalContent>
+    <AdditionalContent Include="$(RazorSdkRoot)Targets\**\*.*">
+      <Pack>true</Pack>
+      <PackagePath>build\netstandard2.0</PackagePath>
+    </AdditionalContent>
+    <AdditionalContent Include="$(RazorSdkRoot)buildMultiTargeting\**\*.*">
```

Comment:
Feel free to remove this + the `build` directory that contain targets . We're no longer shipping this as a NuGet package so they're never used.